app/frontend/src/features/profile/HostingPreferenceForm.tsx

```diff
@@ -1,10 +1,234 @@
-import { makeStyles } from "@material-ui/core";
-import React from "react";
+import {
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import { Autocomplete } from "@material-ui/lab";
```

Comment:
Intentional as our current `Autocomplete` wrapper is too inflexible right now and it's generally a bad idea to tag on too many props just to drill it through to the text input component. It's easier to just use the original Autocomplete in this case.